docs/api/farming.md

```diff
@@ -156,16 +159,16 @@ http://localhost:1317/cosmos/farming/v1beta1/plans/1
 }
 ```
 
-### Stakings
+### Position
 
-Query for all stakings by a farmer: 
+Query for farming position of a farmer:
 
 
 Example Request 
 
 <!-- markdown-link-check-disable -->
 ```bash
-http://localhost:1317/cosmos/farming/v1beta1/stakings/cosmos185fflsvwrz0cx46w6qada7mdy92m6kx4gqx0ny
+http://localhost:1317/squad/farming/v1beta1/position/cosmos185fflsvwrz0cx46w6qada7mdy92m6kx4gqx0ny
```

Comment:
Is there a reason why you used `position` instead of `positions`?

Reply:
Since there's no concept of a collection of `Position`. It's similar to an API like `/profile/{user}`.